example/cumulus-tf/cumulus_distribution.tf

```diff
@@ -8,19 +23,21 @@ module "cumulus_distribution" {
   prefix                   = var.prefix
   api_gateway_stage        = local.distribution_api_gateway_stage
   api_url                  = var.cumulus_distribution_url
+  bucket_map_file          = aws_s3_bucket_object.bucket_map_yaml_distribution.id
+  bucketname_prefix        = ""
+  cmr_acl_based_credentials = true
+  cmr_environment           = var.cmr_environment
+  cmr_provider              = var.cmr_provider
   lambda_subnet_ids        = var.lambda_subnet_ids
   oauth_client_id          = var.csdap_client_id
```

Comment:
Lauren wanted to change the csdap names (csdap_client_id) because others will be using it. 
Maybe just oauth_client_id ?

Reply:
Talked about this 👍